app/code/Magento/PageCache/view/frontend/web/js/page-cache.js

```diff
@@ -53,7 +53,22 @@ define([
                 }
             }
 
-            $(element).contents().each(function (index, el) {
+            // rewrite jQuery contents()
+             var contents = function (element) {
+                return $.map(element, function (elem) {
+                    try {
+                        return $.nodeName(elem, "iframe") ?
+                               elem.contentDocument || (elem.contentWindow ? elem.contentWindow.document : []) :
+                               $.merge([], elem.childNodes);
+                    } catch (e) {
+                        return [];
```

Comment:
Please add logging of this exception as "warning" using "Magento/Ui/view/base/web/js/lib/logger/console-logger.js"